examples/single-sign-on/nile/Server.ts

```diff
@@ -4,7 +4,7 @@ const nile = Server({
   workspace: String(process.env.WORKSPACE),
   database: String(process.env.DATABASE),
   api: {
-    basePath: 'http://localhost:8080',
+    basePath: String(process.env.NILE_BASE_PATH),
```

Comment:
I think this will only be for us locally testing it, vs what someone will actually do when they run this (which will be point at production/whatever our default is)